pkgs/development/ocaml-modules/bistro/default.nix

```diff
@@ -15,25 +16,28 @@
 
 buildDunePackage rec {
   pname = "bistro";
-  version = "unstable-2021-07-13";
+  version = "unstable-2021-11-13";
 
   useDune2 = true;
 
   src = fetchFromGitHub {
     owner = "pveber";
     repo = pname;
-    rev = "4ce8d98f34f15ebf63ececccc9c763fec2b5fa6d";
-    sha256 = "sha256:16vxcdsj4dmswgm6igshs3hirz8jrg8l5b2xgcnxxgvsrc9sxljs";
+    rev = "fb285b2c6d8adccda3c71e2293bceb01febd6624";
+    sha256 = "sha256-JChDU1WH8W9Czkppx9SHiVIu9/7QFWJy2A89oksp0Ek=";
   };
 
-  # Fix build with ppxlib 0.23
-  postPatch = ''
-    substituteInPlace ppx/bistro_script.ml \
-      --replace 'Parser.parse_expression' 'Ocaml_common.Parser.parse_expression'
-  '';
-
   propagatedBuildInputs = [
-    base64 bos core lwt_react ocamlgraph ppx_sexp_conv rresult sexplib tyxml
+    base64
+    bos
+    core
+    lwt_react
+    ocamlgraph
+    ppxlib
```

Comment:
Why this change? I think `ppxlib` should not be propagated.

Reply:
Reverted - upstream added it to their dependencies but I agree it probably doesn't make sense to propagate.